pkg/repoowners/repoowners.go

```diff
@@ -663,14 +663,49 @@ func ParseAliasesConfig(b []byte) (RepoAliases, error) {
 	result := make(RepoAliases)
 
 	config := &struct {
-		Data map[string][]string `json:"aliases,omitempty"`
+		Data map[string]interface{} `json:"aliases,omitempty"`
 	}{}
 	if err := yaml.Unmarshal(b, config); err != nil {
 		return result, err
 	}
 
 	for alias, expanded := range config.Data {
-		result[github.NormLogin(alias)] = NormLogins(expanded)
+		switch v := expanded.(type) {
+		case []interface{}:
+			// Convert []interface{} to []string
+			var members []string
+			for _, member := range v {
+				memberAsString, ok := member.(string)
+				if !ok {
+					return result, fmt.Errorf("unexpected type for alias group member: %T", member)
+				}
+				members = append(members, memberAsString)
+			}
+			if len(members) == 0 {
+				return result, fmt.Errorf("alias group '%s' is empty", alias)
+			}
+			result[github.NormLogin(alias)] = NormLogins(members)
+		case string:
+			// Handle empty string as an empty list
+			if v == "" {
+				return result, fmt.Errorf("alias group '%s' is empty", alias)
+			}
```

Comment:
we should never see a non-empty string here, right? If so can we hint code readers about that with a godoc, and/or maybe even either remove the conditional or add a "weird thing should not happen" error return?